### PR TITLE
General enhancements to Mincer.Logger

### PR DESCRIPTION
**Pull request by jisaacks**
The commit messages sum things up.

Also, I noticed with latest version of node-sass breaks the tests. Had to install an older version (`1.2.3`) to get them to work. Spent a few minutes trying to figure out why but not familiar enough with the code base and didn't want to keep digging.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,18 +42,19 @@
...
     "csso": "*",
     "csswring": "*",
     "ejs": "*",
+    "eslint": "0.10.2",
+    "eslint-plugin-nodeca": "~1.0.3",
     "jade": "*",
     "less": "*",
+    "mocha": "*",
     "node-sass": "*",
+    "sinon": "^1.12.2",
```
Comment (ixti) [inline package.json]: :+1:

**Comment by ixti** [comment]
LGTM! I have only couple of humble requests, please:

- squash PR into a single commit.
- rebase PR on top of master (I have fixed some things in there, so Travis-CI build should pass)

**Comment by jisaacks** [comment]
@ixti done. Thanks.

**Comment by ixti** [comment]
Thanks!

**Pull request merged**